### PR TITLE
feature (CORE-167): add website, heroUrl and index fields for Dapp

### PR DESCRIPTION
**Pull request by Kiryous**
Related PRs
- https://github.com/dialectlabs/sdk/pull/70

TODO
- [ ] Add some tests

### CODE REVIEW

**Review by tsmbl — CHANGES_REQUESTED** [review]
Awesome! Added several comments

File: prisma/schema.prisma [hunk 1]
```diff
@@ -68,7 +68,10 @@ model Dapp {
   id          String  @id @default(uuid()) @db.Uuid
   name        String  @map("name")
   description String? @map("description")
+  index Int? @map("index")
+  website String? @map("website")
   avatarUrl   String? @map("avatar_url")
+  heroUrl   String? @map("hero_url")
```
[the remark this answers is not in the document]
Comment (Kiryous) [reply prisma/schema.prisma:74]: make sense, thx!

File: src/dapp/dapp.controller.dto.ts [hunk 2]
```diff
@@ -48,7 +55,16 @@ export class CreateDappCommandDto {
   readonly description?: string;
   @IsString()
   @IsOptional()
+  readonly website?: string;
+  @IsString()
+  @IsOptional()
   readonly avatarUrl?: string;
+  @IsString()
```
[the remark this answers is not in the document]
Comment (Kiryous) [reply src/dapp/dapp.controller.dto.ts:62]: there is, added

File: prisma/schema.prisma [hunk 3]
```diff
@@ -68,7 +68,10 @@ model Dapp {
   id          String  @id @default(uuid()) @db.Uuid
```
[the remark this answers is not in the document]
Comment (Kiryous) [reply prisma/schema.prisma:68]: done!

[the remark this answers is not in the document]
Comment (Kiryous) [reply prisma/schema.prisma:68]: added, nice catch!

File: src/dapp/dapp.controller.dto.ts [hunk 4]
```diff
@@ -22,8 +27,11 @@ export class DappDto {
     return {
       id: dapp.id,
       name: dapp.name,
-      ...(dapp.avatarUrl && { avatarUrl: dapp.avatarUrl }),
       ...(dapp.description && { description: dapp.description }),
+      ...(dapp.websiteUrl && { websiteUrl: dapp.websiteUrl }),
+      ...(dapp.avatarUrl && { avatarUrl: dapp.avatarUrl }),
+      ...(dapp.heroUrl && { heroUrl: dapp.heroUrl }),
+      ...(dapp.index && { index: dapp.index }),
```
Comment (tsmbl) [inline src/dapp/dapp.controller.dto.ts:34]: let's omit this field in API, wdyt?

**Review by tsmbl — CHANGES_REQUESTED** [review]
Overall lgtm, small comment, lmk wdyt

File: src/dapp/dapp.controller.dto.ts (hunk 4 above)
Comment (Kiryous) [reply src/dapp/dapp.controller.dto.ts]: agreed!

**Review by tsmbl — APPROVED** [review]

**Pull request merged**